inference/core/workflows/execution_engine/dynamic_blocs/block_scaffolding.py

```diff
@@ -0,0 +1,71 @@
+import types
+from typing import List, Type
+
+from inference.core.workflows.execution_engine.dynamic_blocs.entities import PythonCode
+from inference.core.workflows.prototypes.block import (
+    BlockResult,
+    WorkflowBlock,
+    WorkflowBlockManifest,
+)
+
+IMPORTS_LINES = [
```

Comment:
one may import more in the string with python code, like:

```python
import cv2
# any other import 

def function(x, y, z):
    pass
```

we could also extend the imports, yet for instance `CLIP` should probably be used via model manager as manual init in multiple blocks may end up OOM on multiple instances on the same model. We can obviously add more imports which I will do, the question is what we suggest people regarding model usage - I believe it would be better to suggest using models within existing blocks if possible and only use model in-place when there is no other option

Reply:
ok, added separate field in manifest to declare optional imports - that is going to give us more visibility